.bazelrc

```diff
@@ -49,12 +41,16 @@ test --build_tests_only
 
 # Show all the problems
 test --test_output=errors
+build --verbose_failures
```

Comment:
IMO, not related to this PR and can be removed.

Reply:
removed